test/presenters/return-requirements/setup.presenter.test.js

```diff
@@ -34,41 +34,92 @@ describe('Return Requirements - Setup presenter', () => {
 
   describe('when provided with a session', () => {
     it('correctly presents the data', () => {
-      const result = SetupPresenter.go(session)
+      const result = SetupPresenter.go(session, [])
 
       expect(result).to.equal({
         backLink: '/system/return-requirements/61e07498-f309-4829-96a9-72084a54996d/reason',
         licenceRef: '01/ABC',
         sessionId: '61e07498-f309-4829-96a9-72084a54996d',
+        radioOptions: [{
+          checked: false,
+          text: 'Start by using abstraction data',
+          value: 'use-abstraction-data'
+        }, {
+          divider: 'or'
+        }, {
+          checked: false,
+          text: 'Set up manually',
+          value: 'set-up-manually'
+        }],
         setup: null
       })
     })
   })
 
   describe("the 'backLink' property", () => {
     it("returns a link back to the 'start-date' page", () => {
-      const result = SetupPresenter.go(session)
+      const result = SetupPresenter.go(session, [])
 
       expect(result.backLink).to.equal('/system/return-requirements/61e07498-f309-4829-96a9-72084a54996d/reason')
     })
   })
 
+  describe("the 'radioOptions' property", () => {
```

Comment:
We recently changed the [convention](https://github.com/DEFRA/water-abstraction-team/pull/119) when quoting something in a sentence to use double quotes inside and singles outside. 
```suggestion
  describe('the "radioOptions" property', () => {
```
Jason is working on the PR to change all the current files for this at the moment.